spec/javascript/sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
+    return a + b;
+}
```

Comment:
Out of curiosity, why did you abstract this into its own function?

Reply:
This has since been removed and was just a primitive test.